### ISSUE TITLE
Feature Request: Add different format/rotation card pools to the API.

### ISSUE THREAD

**Issue by bcsj** [Null-Signal-Games/netrunnerdb#467]
As discussed a bit on the Slack it would be really nice to be able to pull for instant the "standard card pool" through the API. Maybe optionally with or without the banned cards even.

**Comment by truefalsename** [comment]
Partially on this topic, would it be possible to add format filters when we are prompted to select a corp or  runner ID for a new deck? Just so the page could show only the IDs legal in the chosen format.

Edit
And, it would be great if the format selection would carry over to the build page, so that the card pool would already be set accordingly.

**Comment by plural** [comment]
yeah, indeed.  Right now NRDB doesn't *really* have a notion of format, so we have some stuff to work out before we support this.  Things are implicitly standard right now.
